Source/WebCore/Modules/speech/SpeechRecognitionCaptureSourceImpl.cpp

```diff
@@ -40,7 +41,7 @@ namespace WebCore {
 #if !RELEASE_LOG_DISABLED
 static const void* nextLogIdentifier()
 {
-    static uint64_t logIdentifier = cryptographicallyRandomNumber();
+    static uint64_t logIdentifier = cryptographicallyRandomNumber<uint32_t>();
```

Comment:
Left this as is, generating a 32-bit random number and storing it in a 64-bit global. New function name makes that a lot more obvious. Not sure if it’s a mistake.